### PR TITLE
Correct Fortigate JSON source

### PR DESCRIPTION
**Pull request by blaise-sumo**
The source had a misspelled symbol ($$Foritgate) @carlsalazar1  requested that I open a PR.

### CODE REVIEW

File: Amazon_Web_Services/RDS/Log-Collection/AWS_RDS_Logs_Metrics.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
...
+      "queryString":"$$Cloudtrail\"rds.amazonaws.com\"\n| json \"userIdentity.userName\" as userName nodrop\n| json \"eventName\",\"eventSource\",\"userAgent\",\"awsRegion\",\"eventType\",\"sourceIPAddress\" nodrop\n| json field=_raw \"requestParameters.logFileName\", \"requestParameters.dBInstanceIdentifier\", \"requestParameters.numberOfLines\" as logFileName,dBInstanceIdentifier,numberOfLines nodrop\n| where eventSource matches \"rds.amazonaws.com\"\n| lookup latitude, longitude, country_code, country_name, region, city, postal_code, area_code, metro_code from geo://default on ip=sourceIPAddress\n| count by latitude, longitude, country_code, country_name, region, city, postal_code, area_code, metro_code\n| sort _count\n\n",
```
Comment (kb18951452) [inline Amazon_Web_Services/RDS/Log-Collection/AWS_RDS_Logs_Metrics.json]: I think this lacks the trailing space between the cloudtrai* and \"rds.amaz...." on the first line. 

Can you verify that AFTER following the readme, that the find/replace works?

Comment (kb18951452) [reply Amazon_Web_Services/RDS/Log-Collection/AWS_RDS_Logs_Metrics.json]: Also, if you don't mind me asking, why are there so many commits as part of this PR?

Comment (blaise-sumo) [reply Amazon_Web_Services/RDS/Log-Collection/AWS_RDS_Logs_Metrics.json]: Wow! @kb18951452 There are a lot of commits I don't recognize here!! In fact, I only made https://github.com/SumoLogic/sumologic-content/pull/69/commits/ae9134df9c12909dd7e0849ff16f75424a4b6feb

Let's back off on this and figure out why those other commits never made it in before.

**Comment by wjakelee** [comment]
A lot of this content is either no longer relevant, has been added to app catalog, or been migrated to new form

**Pull request closed without merging**